Update Dapr extension for AKS document to add details for managing Dapr versioning (#116125)
Azure/azure-content: articles/aks/dapr-settings.md

   - `NO_PROXY`
1. [Installing the proxy certificate in the sidecar](https://docs.dapr.io/operations/configuration/install-certificates/).

## Updating your Dapr installation version

If you are on a specific Dapr version and you don't have `--auto-upgrade-minor-version` available, you can use the following command to upgrade or downgrade Dapr:

```azurecli
az k8s-extension update --cluster-type managedClusters \
--cluster-name myAKSCluster \
--resource-group myResourceGroup \
--name dapr \
--version 1.12.0 # Version to upgrade or downgrade to
```

> **Note** that this will update the Dapr control plane only, restart your application deployments to update the Dapr sidecars:

Reviewer: ```suggestion
The above command updates the Dapr control plane **only.** To update the Dapr sidecars, restart your application deployments:
```

> The above command updates the Dapr control plane **only.** To update the Dapr sidecars, restart your application deployments:

Reviewer 2: ```suggestion
The preceding command updates the Dapr control plane *only.* To update the Dapr sidecars, restart your application deployments:
```
Edits for Microsoft style
